### DISCUSSION TITLE
This forum has an RSS feed.

### DISCUSSION THREAD

**Discussion by vincode-io** [vincode-io/Zavala#221] [General]
If you want to keep up with Zavala Discussions subscribe to the feed below in your RSS reader of choice.

https://github.com/vincode-io/Zavala/discussions.atom

If you need an RSS reader, [NetNewsWire](https://netnewswire.com) is a free and open source reader that is well respected in the Apple community.

If you aren't familiar with what RSS is, I think this help document from [NetNewsWire](https://netnewswire.com) is pretty good: [What is RSS? What are feeds?](https://netnewswire.com/help/mac/6.1/en/what-is-rss.html)

**Comment by Khauswest** [comment]
I am impressed with this outliner - so simple and intuitive. But I am having an issue.
I have changed the fonts in the settings to the Mac system font Bookman Oldstyle. The outline displays the Bookman font correctly on screen, but when I print the document, all of the text prints as Helvetica. Is this normal behavior or am I missing something?

**Reply by vincode-io** [reply]
Thanks for creating https://github.com/vincode-io/Zavala/issues/280. Let's discuss there.
